### PR TITLE
Add templates for the user-feedback repo

### PR DESCRIPTION
**Pull request by mihaiep**
PR to add the templates for the user-feedback repo; [Templates  ](https://github.com/mhdawson/create-node-meeting-artifacts/tree/master/templates)

### CODE REVIEW

File: meetings/2018-02-16-Public-meeting.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+* Intro 
+* **Michael Dawson: Introduction to the Benchmarking WG.**
+   * What the Benchmarking Group is?  
+   * Repo here: https://github.com/nodejs/benchmarking
+    The focus: we want to track and evangelize performance gains
+    between releases and avoid performance regressions. 
+    Make sure Node is running efficiently.
+1.  Define Use cases: https://github.com/nodejs/benchmarking/blob/master/docs/use_cases.md
+ 
+If you have a user case that doesn’t fit into one of these here is important for you to let us know,
```
Comment (mhdawson) [inline meetings/2018-02-16-Public-meeting.md]: use case

File: meetings/2018-02-16-Public-meeting.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+ 
+If you have a user case that doesn’t fit into one of these here is important for you to let us know, 
+so we can factor in additional use cases into the work we do here.
+ 
+2. Key runtime attributes
+https://github.com/nodejs/benchmarking/blob/master/docs/runtime_attributes.md
+ 
+3. Case coverage (coverage of use cases and key attributes)-to track performance
+https://github.com/nodejs/benchmarking/blob/master/docs/case_coverage.md.
+The benches running every night https://benchmarking.nodejs.org/.
```
Comment (mhdawson) [inline meetings/2018-02-16-Public-meeting.md]: benchmarks

**Comment by mihaiep** [comment]
@mhdawson, Michael the file [above](meetings/2018-02-16-Public-meeting.md) should be in PR [#33](https://github.com/nodejs/user-feedback/pull/33) and not here.

Initially this PR, was opened for  adding templates for the user-feedback repo, but I had no clu now how to do that. Can you help on this?   Thanks

**Comment by mhdawson** [comment]
@mihaiep to add the templates you want to open a PR in this repo: https://github.com/mhdawson/create-node-meeting-artifacts

and create files in the https://github.com/mhdawson/create-node-meeting-artifacts/tree/master/templates directory. You can pick one of the existing ones (ex commcomm) and copy the templates (4 files) *_commcomm and then edit as per: https://github.com/mhdawson/create-node-meeting-artifacts/blob/master/TEMPLATES_DOCUMENTATION.md.

If you can take a first cut and open a PR I can then test and fixup to get them working.

**Comment by dshaw** [comment]
@mihaiep @mhdawson Do we have a path to resolving the issues here?

**Comment by mihaiep** [comment]
I'm in a process to clean all these old commits that shouldn't be here, then I'll dive into this.

**Comment by mihaiep** [comment]
Closing this, and open a new PR in Michael's repo.

**Pull request closed without merging**